### PR TITLE
Fixes few reagent runtimes.

### PR DESCRIPTION
**Pull request by AnturK**

### CODE REVIEW

**Review by ShizCalev — CHANGES_REQUESTED** [review]
image


Got a few more spots with this issue still.

File: code/modules/food_and_drinks/food/snacks_pastry.dm [hunk 1]
```diff
@@ -54,7 +54,7 @@
 /obj/item/reagent_containers/food/snacks/donut/chaos/Initialize()
 	. = ..()
 	extra_reagent = pick(/datum/reagent/consumable/nutriment, /datum/reagent/consumable/capsaicin, /datum/reagent/consumable/frostoil, /datum/reagent/drug/krokodil, /datum/reagent/toxin/plasma, /datum/reagent/consumable/coco, /datum/reagent/toxin/slimejelly, /datum/reagent/consumable/banana, /datum/reagent/consumable/berryjuice, /datum/reagent/medicine/omnizine)
-	reagents.add_reagent("[extra_reagent]", 3)
+	reagents.add_reagent(extra_reagent, 3)
 
 /obj/item/reagent_containers/food/snacks/donut/jelly
 	name = "jelly donut"
```
Comment (ShizCalev) [inline code/modules/food_and_drinks/food/snacks_pastry.dm]: including the one right below this.

Comment (AnturK) [reply code/modules/food_and_drinks/food/snacks_pastry.dm]: I'll go over them

**Comment by AnturK** [comment]
https://github.com/tgstation/tgstation/blob/7fab3e29084777c2315b0b4184ec6e36e03f1650/code/datums/dna.dm#L591 This reagent never existed so no idea what the author had in mind here. Slime ? Lizard ?

File: code/modules/awaymissions/mission_code/snowdin.dm [hunk 2]
```diff
@@ -171,7 +171,7 @@
 	if(C.reagents.total_volume >= C.volume)
 		to_chat(user, "<span class='danger'>[C] is full.</span>")
 		return
-	C.reagents.add_reagent("plasma", rand(5, 10))
+	C.reagents.add_reagent(/datum/reagent/stable_plasma, rand(5, 10))
```
Comment (Tlaltecuhtli) [inline code/modules/awaymissions/mission_code/snowdin.dm]:
```suggestion
	C.reagents.add_reagent(/datum/reagent/toxin/plasma, rand(5, 10))
```

File: code/datums/dna.dm [hunk 3]
```diff
@@ -569,7 +569,7 @@
 				physiology.damage_resistance = -20000
 			if(5)
 				to_chat(src, "<span class='notice'>Oh, I actually feel quite alright!</span>")
-				reagents.add_reagent("mutationtoxin2", 10)
+				reagents.add_reagent(/datum/reagent/slime_toxin, 10)
```
Comment (ShizCalev) [inline code/datums/dna.dm]: /datum/reagent/aslimetoxin

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Review by ShizCalev — APPROVED** [review]

**Pull request merged**